### ISSUE TITLE
(Self Voicing) Name of mass spinner is not read on focus

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/gravity-force-lab-basics#217]
Doing a review, I am seeing that the name of the mass spinner is not being read upon keyboard focus. It should be according to the design document.

**Comment by jessegreenberg** [comment]
Ah, this is caused by changes made for #213 and #214.

**Comment by jessegreenberg** [comment]
Fixed in the above commit.
